general.py

```diff
@@ -0,0 +1,54 @@
+def addInts(intArray):
+    total = 0
+
+    for integer in intArray:

```

Comment:
You need to check if theyre even first 

Also this could be done as a list comprehension to speed things up